docs/Contributing/documentation_guidelines.md

```diff
@@ -122,26 +122,32 @@ Next, make sure you have `Ruby` installed; the best way to check this is to do `
 To install ruby, just do:
 
 ```
-brew install ruby
+brew install ruby@3.2
```

Comment:
Did this work for you?

Reply:
BTW I rename this file to `docs/documentation/documentation_guidelines.md` in #79, so we'll have conflicts to deal with.

Reply:
okay, makes sense - yes that worked for me!